packages/base-driver/lib/basedriver/core.ts

```diff
@@ -308,11 +316,15 @@ export class DriverCore<const C extends Constraints, Settings extends StringReco
 
     if (!_.includes(validStrategies, strategy)) {
       throw new errors.InvalidSelectorError(
-        `Locator Strategy '${strategy}' is not supported for this session`
+        `Locator Strategy '${strategy}' is not supported for this session`,
       );
     }
   }
 
+  get bidiProxyUrl(): string | null {
```

Comment:
a little doctring in the core class might be useful

Reply:
yep good idea